torchtext/data/utils.py

```diff
@@ -6,7 +6,7 @@
 from functools import partial
 
 
-def _split_tokenizer(x):
+def _split_tokenizer(x): # noqa: F821
     # type: (str) -> List[str]
```

Comment:
Since we only support python3 now, we can use python3 type annotation here.

Reply:
This lint is still being raised internally due to some issues in flake8. This way we suppress the error.